src/main/java/net/fabricmc/loom/build/MixinRefmapHelper.java

```diff
@@ -94,6 +119,26 @@ public static boolean addRefmapName(Project project, Path outputPath) {
 		}
 	}
 
+	private static String[] getRootPaths(SourceSet sourceSet) {
+		Set<Fire> dirs = ;
```

Comment:
This doesnt compile.

Reply:
Oops, I guess I forgot to save before pushing :/